test_scripts/Capabilities/UpdateVideoStreamingCapabilities/OnSystemCapabilityUpdated/011_OnSystemCapabilityUpdated_multiple_apps.lua

```diff
@@ -0,0 +1,132 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description: Processing of OnSystemCapabilityUpdated notification for 2 applications at once
+--
+-- Preconditions:
+-- 1. HMI capabilities contain data about videoStreamingCapability
+-- 2. SDL and HMI are started
+-- 3. App1 and App2 are registered and activated
+--
+-- Sequence:
+-- 1. HMI sends OnSystemCapabilityUpdated notification for "VIDEO_STREAMING" to SDL
+-- SDL does:
+--  a. not send OnSystemCapabilityUpdated (videoStreamingCapability) notification to App1 and App2
+-- 2. App1 subscribes on videoStreamingCapability updates
+-- 3. HMI sends OnSystemCapabilityUpdated notification for "VIDEO_STREAMING" to SDL
+-- SDL does:
+--  a. send OnSystemCapabilityUpdated (videoStreamingCapability) notification to App1
+--   and doesn't send notification to App2
+-- 4. App2 subscribes on videoStreamingCapability updates
+-- 5. HMI sends OnSystemCapabilityUpdated notification for "VIDEO_STREAMING" to SDL
+-- SDL does:
+--  a. send OnSystemCapabilityUpdated (videoStreamingCapability) notification to each apps
+-- 6. App1 unsubscribes from videoStreamingCapability updates
+-- 7. HMI sends OnSystemCapabilityUpdated notification for "VIDEO_STREAMING" to SDL
+-- SDL does:
+--  a. send OnSystemCapabilityUpdated (videoStreamingCapability) notification to App2
+--   and doesn't send notification to App1
+-- 8. App2 unsubscribes from videoStreamingCapability updates
+-- 9. HMI sends OnSystemCapabilityUpdated notification for "VIDEO_STREAMING" to SDL
+-- SDL does:
+--  a. not send OnSystemCapabilityUpdated (videoStreamingCapability) notification to each apps
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/Capabilities/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionIdAbsent = nil
+local appSessionId1 = 1
+local appSessionId2 = 2
+local expected = 1
+local notExpected = 0
+local isSubscribe = true
+local isUnSubscribe = false
+
+--[[ Local Functions ]]
+local function sendOnSystemCapabilityUpdatedMultipleApps(pAppId, pTimesAppId1, pTimesAppId2)
+  local mobileParams = {
+    systemCapability = {
+      systemCapabilityType = "VIDEO_STREAMING",
+      videoStreamingCapability = common.buildVideoStreamingCapabilities()
+    }
+  }
+  local hmiParams = common.cloneTable(mobileParams)
+  if pAppId then
+    hmiParams.appID = common.getHMIAppId(pAppId)
+  end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", hmiParams)
```

Comment:
Add sending OnSystemCapabilityUpdated to both apps each time.

Reply:
In each case, the script checks a sending of OnSystemCapabilityUpdated with appId1, appId2, and without appId.